### DISCUSSION TITLE
Setup fails, please see the attached pdf of sosetup.log, my VMnets and all the parms i filled out before choosing the final yes in setup

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#4974] [General]
This attatchmemnt is most of the log, VMnet setup and the print of the parameters i entered. if something jumps out at you, please let me know.
   At have been at this for 4 days. 
   thanks for any suggestions or advice 
 
[security onion sosetup2.log.pdf](https://github.com/Security-Onion-Solutions/securityonion/files/6896703/security.onion.sosetup2.log.pdf)

**Comment by TotieBash** [comment]
It looks like its network connectivity.  Second page of the pdf "Management IP:  192.18.80.46", I think the second octet you meant x.168.x.x.

**Reply by iqworks** [reply]
thank you, i will double check that.

**Reply by iqworks** [reply]
i ran setup again, and this is the sosetup.log now. I also think it might be a connection problem because of all the "cannot resolve" messages. ALSO, not sure about what IP to enter in the Management IP, Gateway and Monitor propmpts. Cannot find documentation on what IPs to enter.
thanks again for taking a look. Please let me know if you see anything else advice.   

[security onion sosetup.logDD.pdf](https://github.com/Security-Onion-Solutions/securityonion/files/6900535/security.onion.sosetup.logDD.pdf)

**Reply by TotieBash** [reply]
yeah it looks like still ip connection problem but you have two options.  On page 1 screenshot, make sure your VMware workstation "VMnet4" (assuming that is the Management int) make sure it is set to "bridge".  I assume that might be your issue.  you should be able to ping 192.168.50.44 from your vm host machine...

Second solution is during installation the first or second question choose "air gapped" network setup.  That should work to complete the install...  You still need the network issue though

**Comment by iqworks** [comment]
Hi TotieBash, hope all is well. Thank you for the advice. YEA , looks like it installed. I added the bridge network and selected airgap as you suggested. 

 
I am sending you the new PDF. Now, as you can see if have new issues, plus looking at the network. I cannot ping to 192.168.50.44.
C:\WINDOWS\system32>ping 192.168.50.44

Pinging 192.168.50.44 with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 192.168.50.44:
    Packets: Sent = 4, Received = 0, Lost = 4 (100% loss),

C:\WINDOWS\system32>^Z^Z

Thanks so much !! you are the person (I don’t know if you are male or female 😊)
(hope you get this email reply. I will post this on the forum as answer in case you get credit). 
Please, feel free to comment on this PDF if you have time. 


[security onion sosetup.logEE.pdf](https://github.com/Security-Onion-Solutions/securityonion/files/6903563/security.onion.sosetup.logEE.pdf)

**Reply by iqworks** [reply]
Hi, please see this one instead, i documented a few more things. 
[VM Security Onion - iq SO STENODO 2.pdf](https://github.com/Security-Onion-Solutions/securityonion/files/6904329/VM.Security.Onion.-.iq.SO.STENODO.2.pdf)
